jinad/api/endpoints/pod.py

```diff
@@ -0,0 +1,176 @@
+import uuid
+from argparse import Namespace
+from typing import Dict
+from fastapi import status, APIRouter, Body, Response
+from fastapi.responses import StreamingResponse
+from logger import get_logger
+
+from jina.peapods.pod import MutablePod
+from jina.enums import PeaRoleType, SchedulerType, OnErrorSkip, SocketType
+
+from models.pod import PodModel, RemotePodModel
+from config import openapitags_config
+from excepts import HTTPException
+
+logger = get_logger(context='pod-context')
+router = APIRouter()
+pod_dict = {}
+TAG = openapitags_config.POD_API_TAGS[0]['name']
+
+
+@router.on_event('startup')
+async def startup():
+    logger.info('Welcome to Jina daemon for remote pods')
+
+
+@router.put(
+    path='/pod',
+    summary='Create a MutablePod locally (triggered via a Flow)',
+    tags=[TAG]
+)
+async def _create(
+    pod_arguments: Dict
+):
+    """Used to create a MutablePod on localhost!
+    This is usually a remote pod which gets triggered by a Flow context
+    Shouldn't be created with manual trigger 
+
+    Args:
+        pod_arguments (Dict): accepts Pod args
+    """
+    global pod_dict
```

Comment:
why does it need a `global` pod_dict?

Reply:
Is this because these are `Pods` that are not under the control of the `Flow` right?

Reply:
`Pod` is still under `Flow` control, but it is invoked using an api endpoint, which receives the `pod_id`. In case the server is shutdown, we need a logic to kill the pod process, that's where this comes in use

Reply:
who is calling this endpoint?